reactquiz/src/components/quiz/Results.js

```diff
@@ -1,34 +1,25 @@
-import React,{Component} from 'react';
-
-
-class Results extends Component{
-
+import React, { Component } from 'react';
 
+class Results extends Component {
 	render() {
-
-		var percent =(this.props.score / this.props.questions.length*100);
-		if (percent > 80){
-
-			var message ='Awesome Job!';
-		} else if(percent < 80 && percent >60)
-		{
-			var message = 'You did Ok !';
-		}
-		else {
-
-			var message ='Not satisfactory. Should try once again !';
+		const percent = (this.props.score / this.props.questions.length * 100 * 100);
+		let message;
+		if (percent > 80) {
+			message = 'Awesome Job!!';
+		} else if (percent > 60) {
+			message = 'You did Ok!';
+		} else {
+			message = 'Not satisfactory. Should try once again!!!';
 		}
-		return(
+		return (
 			<div className="well">
-				<h4> You got {this.props.score} out of {this.props.questions.length} Correct</h4>
+				<h4>You got {this.props.score} out of {this.props.questions.length} Correct</h4>
```

Comment:
For consistency and better readability, consider using template literals when constructing the output string that includes `percent` and `message`.